hazelcast/src/main/java/com/hazelcast/nio/serialization/GenericRecord.java

```diff
@@ -292,7 +327,7 @@ static Builder portable(ClassDefinition classDefinition) {
          *                                         Same field is trying to be overwritten without using
          *                                         {@link GenericRecord#cloneWithBuilder()}.
          */
-        Builder writeBoolean(String fieldName, boolean value);
+        Builder writeBoolean(@Nonnull String fieldName, boolean value);
```

Comment:
these are still missing :)

Reply:
I just didn't want to add NonNullable for obvious chaining pattern.